Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivityPresenter.java

```diff
@@ -135,6 +135,16 @@ public boolean shouldSave()
     return mShouldSave;
   }
 
+  public void onSerialPort1SettingChanged(MenuTag key, int value)
+  {
+    if (value != 0) // Not disabled
```

Comment:
Sorry, just one more review comment for a thing I missed earlier. For this setting, 0 doesn't mean disabled, it means Dummy. Most likely you want to check that value is not 0 and is not 255.

Reply:
fixed